apps/omg_watcher/test/support/exit_processor/test_helper.ex

```diff
@@ -25,7 +25,7 @@ defmodule OMG.Watcher.ExitProcessor.TestHelper do
 
   require Utxo
 
-  @exit_id 1
+  @exit_id 9876
```

Comment:
It's unlikely that somebody needs to change the value, and it'll be discovered quickly but maybe to be nice can you add a comment saying the value is used by other tests?
Thanks!